### PR TITLE
[Performance] Run Viking Village perf tests for URP and Built-in

### PR DESCRIPTION
**Pull request by sophiaaar**
### Purpose of this PR
* Run performance tests on Viking Village using URP and Built-in

---
### Testing status
* URP https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/performance%252Fviking-village/.yamato%252Fall-vikingvillage_urp.yml%2523PR_VikingVillage_URP_trunk/5725650/job/pipeline
* Built-in https://yamato.cds.internal.unity3d.com/jobs/902-Graphics/tree/performance%252Fviking-village/.yamato%252Fall-vikingvillage_builtin.yml%2523PR_VikingVillage_BuiltIn_trunk/5725665/job/pipeline

---
### Comments to reviewers
I have started working on the dashboard, but I want to gather some data from running this in nightly before I share it.

### CODE REVIEW

File: .yamato/config/vv_perf_builtin.metafile [hunk 1]
```diff
@@ -0,0 +1,255 @@
...
+        model: rtx2080
+      standalone_build:
+        type: Unity::VM
+        image: sdet/gamecode_win10:stable
+        flavor: b1.xlarge
+      performance_build_editmode:
+        type: Unity::VM
+        image: sdet/gamecode_win10:stable
+        flavor: b1.large
+        model: rtx2080
```
Comment (theo-pnv) [inline .yamato/config/vv_perf_builtin.metafile:139]: I think the model is a typo? The agent is not using a GPU

File: .yamato/config/vv_perf_urp.metafile [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+        model: rtx2080
+      standalone_build:
+        type: Unity::VM
+        image: sdet/gamecode_win10:stable
+        flavor: b1.xlarge
+      performance_build_editmode:
+        type: Unity::VM
+        image: sdet/gamecode_win10:stable
+        flavor: b1.large
+        model: rtx2080
```
Comment (theo-pnv) [inline .yamato/config/vv_perf_urp.metafile:143]: Same

**Review by theo-pnv — APPROVED** [review]

File: .yamato/config/vv_perf_builtin.metafile (hunk 1 above)
Comment (sophiaaar) [reply .yamato/config/vv_perf_builtin.metafile:139]: oh, thank you!

**Comment by sophiaaar** [comment]
cancelled package jobs since this PR does not touch the packages

**Pull request merged**